src/operator/skip.ts

```diff
@@ -8,8 +8,8 @@ import { TeardownLogic } from '../Subscription';
  *
  * <img src="./img/skip.png" width="100%">
  *
- * @param {Number} the `n` of times, items emitted by source Observable should be skipped.
- * @return {Observable} an Observable that skips values emitted by the source Observable.
+ * @param {Number} n - The number of times, items emitted by source Observable should be skipped.
```

Comment:
I'm refering to this https://esdoc.org/manual/usage/tags.html#-param. The `<name>` has to match a parameter name. Function `skip()` has one parameter called `total` not `n`. I think the variable should be actually called `count` to be consistent with [`take()`](https://github.com/ReactiveX/rxjs/blob/master/src/operator/take.ts#L9).